### PR TITLE
added icon to CSV using assets/icon/red-hat-logo.svg as the source

### PR DESCRIPTION
**Pull request by pedjak**
### Motivation

Improving UX

### Changes

Added icon to CSV using assets/icon/red-hat-logo.svg as the source

### Testing

`make push-to-manifest-repo` should generate a CSV containing icon encoded
in base64 format. In addition to that its mediatype should be specified to
image/svg

For further more details refer the CONTRIBUTING.md

### CODE REVIEW

File: deploy/olm-catalog/service-binding-operator/0.0.23/service-binding-operator.v0.0.23.clusterserviceversion.yaml [hunk 1]
```diff
@@ -181,8 +184,8 @@ spec:
     - name: Blog post
       url: https://developers.redhat.com/blog/2019/12/19/introducing-the-service-binding-operator
   maintainers:
-    - email: shbose@redhat.com
-      name: Shoubhik Bose
+    - email: sbo-support@redhat.com
```
Comment (pedjak) [inline deploy/olm-catalog/service-binding-operator/0.0.23/service-binding-operator.v0.0.23.clusterserviceversion.yaml]: @sbose78 who can decide what address should be used? I have seen that serverless team uses `serverless-support@redhat.com`, hence this proposal.

**Comment by sbose78** [comment]
Folks can use github to file issues as of now, while we resolve the email address issue. Feel free to go ahead and take this in @pedjak @Avni-Sharma

**Comment by sbose78** [comment]
+ @DhritiShikhar

**Pull request merged**